app/javascript/components/Product/index.tsx

```
@@ -66,795 +66,850 @@ import { useRunOnce } from "$app/components/useRunOnce";
+                        </button>
+                    ) : null}
+                </section>
+            ) : null}
         </section>
-      ) : null}
-    </section>
-  );
+    );
 };
 
 const Review = ({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Review'. Consider to delete it or rename it.

'Review' is defined here:


(lint/suspicious/noRedeclare)